aries-site/src/components/home/Community.js

```diff
@@ -0,0 +1,99 @@
+import React, { useContext } from 'react';
+
+import {
+  Box,
+  Button,
+  Grid,
+  Heading,
+  Paragraph,
+  ResponsiveContext,
+  Text,
+} from 'grommet';
+import { ChatOption, Github, Map, Slack } from 'grommet-icons';
+
+const LinkButton = props => (
+  <Button
+    target="_blank"
+    rel="noreferrer noopener"
+    alignSelf="start"
+    {...props}
+  />
+);
+
+const SlackOption = () => (
+  <Box gap="medium">
+    <Slack size="large" color="text-strong" />
+    <Box>
+      <Heading level={4} margin="none">
```

Comment:
I think all of these should be an h3, since the "Community" section is an h2. May need to adjust sizing. Same goes for the other sections in this file
```suggestion
      <Heading level={3} margin="none">
```

Reply:
Makes sense. Done.